.github/workflows/bindings_php.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [8.1, 8.2]
+        php: [8.1, 8.2, 8.3]
```

Comment:
Our php CI has been disabled for a while, would you like to try enable them back?

Reply:
Tried. Failed again.

Reply:
Please don't change this since we can't test them.

Reply:
right